# Datasets

## Description
The Rscript [`0_DataGeneration.R`](https://github.com/Jojo6297/edame/blob/main/datasets/0_DataGeneration.R) uses the package **curatedMetagenomicData** to obtain the microbial abundance datasets for 4 health states: Healthy [1-4], T2D (type 2 diabetes)[5-8], CRC (Colorectal cancer) [9-12] and IBD (Inflammatory Bowel disease) [13-17]. 

### References
1. Zhernakova A, Kurilshikov A, Bonder MJ, et al. Population-based metagenomics analysis reveals markers for gut microbiome composition and diversity. Science. 2016;352(6285):565-569. https://doi.org/10.1126/science.aad3369
2. Asnicar F, Berry SE, Valdes AM, et al. Microbiome connections with host metabolism and habitual diet from 1,098 deeply phenotyped individuals. Nat Med. 2021;27(2):321-332. https://doi.org/10.1038/s41591-020-01183-8
3. Mehta RS, Abu-Ali GS, Drew DA, et al. Stability of the human faecal microbiome in a cohort of adult men. Nat Microbiol. 2018;3(3):347-355. https://doi.org/10.1038/s41564-017-0096-0
4. Zeevi D, Korem T, Zmora N, et al. Personalized nutrition by prediction of glycemic responses. *Cell.* 2015;163(5):1079-1094. https://doi.org/10.1016/j.cell.2015.11.001

5. Ho KM, Kalgudi S, Corbett J-M, Litton E. Gut microbiota in surgical and critically ill patients. Anaesthesia and Intensive Care. 2020;48(3):179-19. https://doi.org/10.1177/0310057X20903732
6. Qin, J., Li, Y., Cai, Z. et al. A metagenome-wide association study of gut microbiota in type 2 diabetes. Nature 490, 55–60 (2012). https://doi.org/10.1038/nature11450
7. Li, J., Jia, H., Cai, X. et al. An integrated catalog of reference genes in the human gut microbiome. Nat Biotechnol 32, 834–841 (2014). https://doi.org/10.1038/nbt.2942
8. Karlsson, F., Tremaroli, V., Nookaew, I. et al. Gut metagenome in European women with normal, impaired and diabetic glucose control. Nature 498, 99–103 (2013). https://doi.org/10.1038/nature12198
9. Yachida S, Mizutani S, Shiroma H, et al. Metagenomic and metabolomic analyses reveal distinct stage-specific phenotypes of the gut microbiota in colorectal cancer. *Nat Med.* 2019;25(6):968-976. https://doi.org/10.1038/s41591-019-0458-7
10. Wirbel J, Pyl PT, Kartal E, et al. Meta-analysis of fecal metagenomes reveals global microbial signatures that are specific for colorectal cancer. *Nat Med.* 2019;25(4):679-689. https://doi.org/10.1038/s41591-019-0406-6
11. Zeller G, Tap J, Voigt AY, et al. Potential of fecal microbiota for early-stage detection of colorectal cancer. *Mol Syst Biol.* 2014;10(11):766. https://doi.org/10.15252/msb.20145645
12. Vogtmann E, Hua X, Zeller G, et al. Colorectal cancer and the human gut microbiome: reproducibility with whole-genome shotgun sequencing. *PLoS One.* 2016;11(5):e0155362. https://doi.org/10.1371/journal.pone.0155362
13. Schirmer M, Franzosa EA, Lloyd-Price J, et al. Dynamics of metatranscription in the inflammatory bowel disease gut microbiome. *Nat Microbiol.* 2018;3(3):337-346. https://doi.org/10.1038/s41564-017-0089-z
14. Lloyd-Price J, Arze C, Ananthakrishnan AN, et al. Multi-omics of the gut microbial ecosystem in inflammatory bowel diseases. Nature. 2019;569(7758):655-662. https://doi.org/10.1038/s41586-019-1237-9
15. Vich Vila A, Imhann F, Collij V, et al. Gut microbiota composition and functional changes in inflammatory bowel disease and irritable bowel syndrome. Sci Transl Med. 2018;10(472). https://doi.org/10.1126/scitranslmed.aap8914
16. Hall AB, Yassour M, Sauk J, et al. A novel Ruminococcus gnavus clade enriched in inflammatory bowel disease patients. Genome Med. 2017;9:1-12. https://doi.org/10.1186/s13073-017-0490-5
17. Nielsen HB, Almeida M, Sierakowska Juncker A, et al. Identification and assembly of genomes and genetic elements in complex metagenomic samples without using reference genomes. Nat Biotechnol. 2014;32(8):822-828. https://doi.org/10.1038/nbt.2939